### PR TITLE
Rw testing

### LINKED ISSUE #55: Use error in store to display

### LINKED ISSUE #74: Fix Results component to display correct message
-When an invalid/not found search term is used for search, display the error msg that no recipes could be found

-When alcoholic/non-alcoholic drinks lists cannot be found on App page load, display error message (the bar is closed one)

### PR DESCRIPTION
**Pull request by rwilliams659**
### Type of change made:
- [X] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description / What does this PR do?
- Modify Results component to get errorMessage prop from store & then display errorMessage if there is an error, to display a message that there were no results is searchResults is empty array, display welcomeMsg if results are null, and otherwise display actual list of drinks 
- Create resetError action
- Modify Search component to dispatch resetError as part of handleSearch 
- Remove unnecessary lines from DrinkCard & Results tests
- Add additional Results test to ensure it displays error from state if there is one 
- Adds action tests for addRecipeId, createDrinkRecipe, resetError & resetRecipe
- Modify drinksList reducer test to have correct initial state
- Modify errorReducer to include new action case for resetting error & modify test to reflect the change

### Files modified:
- Actions 
- Actions test
- Drink Card test
- Drink Card css
- Results component
- Results test
- Search component
- drinksList reducer test
- errorReducer & test

### Has This Been Tested?
- [ ] No
- [X] Yes

### Are there specific methods for testing?
npm test

### What are the relevant tickets?
resolve #55 
resolve #74 

### Message/Questions for reviewer:
There's a lot here, lmk if you have questions!

### Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/DrinkCard/DrinkCard.css [hunk 1]
```diff
@@ -22,16 +22,8 @@
...
-}
-
-/* .alcoholic {
   background-color: #c3f584;
-} */
-
-/* .non-alcoholic {
-  
-} */
+}
```
Comment (relyt4me) [inline src/components/DrinkCard/DrinkCard.css:26]: We probably need to do another sweep for console logs and comments at some point

File: src/components/Results/Results.js [hunk 2]
```diff
@@ -25,16 +26,20 @@ export const Results = (props) => {
 
   return (
     <>
-      {!resultsList &&
+      {!resultsList && !props.errorMessage && 
         <>
           <h2 className='results-heading'>Welcome to Fridge To Glass!</h2>
-        <p className='welcome-msg'>Please type an ingredient in the search bar above, select your alcohol preference, and click <span className='bold italic'>Find Drinks</span> to see a list of cocktails with that ingredient.<br></br><br></br><span className='bold'>Happy bartending!</span></p>
+          <p className='welcome-msg'>Please type an ingredient in the search bar above, select your alcohol preference, and click <span className='bold italic'>Find Drinks</span> to see a list of cocktails with that ingredient.<br></br><br></br><span className='bold'>Happy bartending!</span></p>
```
Comment (relyt4me) [inline src/components/Results/Results.js:32]: This logic makes sense I like it

File: src/components/Search/Search.js [hunk 3]
```diff
@@ -118,6 +118,7 @@ export const mapDispatchToProps = (dispatch) => {
   return {
     handleSearch: (drinksList) => {
       dispatch(updateDrinksList(drinksList));
+      dispatch(resetError());
```
Comment (relyt4me) [inline src/components/Search/Search.js:121]: Does this mean every time handle search is called both dispatches fire

**Review by relyt4me — APPROVED** [review]

File: src/components/Search/Search.js (hunk 3 above)
Comment (tarynmartin) [reply src/components/Search/Search.js:121]: That's what it looks like to me

Comment (rwilliams659) [reply src/components/Search/Search.js:121]: Yup! Open to suggestions if you think it would work better somewhere else

**Review by tarynmartin — APPROVED** [review]
Tests look great and the conditional logic for displaying messages on the results screen look good.

**Pull request merged**